SWLOR.Game.Server/Item/Medicine/PoisonTreatmentKit.cs

```diff
@@ -28,7 +28,7 @@ public void ApplyEffects(NWCreature user, NWItem item, NWObject target, Location
                 if (_.GetIsEffectValid(effect) == TRUE)
                 {
                     int effectType = _.GetEffectType(effect);
-                    if (effectType == EFFECT_TYPE_POISON || effectType == EFFECT_TYPE_DISEASE)
+                    if (effectType == EFFECT_TYPE_POISON || effectType == EFFECT_TYPE_DISEASE || effectType == EFFECT_TYPE_ABILITY_DECREASE)
```

Comment:
I'm fine with this change. Make sure the item description properly reflects this additional feature.

Reply:
[poisontreatkit.zip](https://github.com/zunath/SWLOR_NWN/files/3682597/poisontreatkit.zip)
Didn't think about that.